doc/jsk_perception/nodes/lidar_person_detection_node.md

```diff
@@ -0,0 +1,79 @@
+# lidar_person_detection_node.py
+
+![](images/lidar_person_detection_node.gif)
+
+Detect person from lidar sensor.
+
+For more detail, please see the thesis `DR-SPAAM: A Spatial-Attention and Auto-regressive Model for Person Detection in 2D Range Data`.
+
+In order to use this feature, you need to install `pytorch <https://pytorch.org/get-started/locally/>`_ (pytorch >= 1.4.0 is recommended).
+
+
+## Subscribing Topic
+
+* `~input` (`sensor_msgs/LaserScan`)
+
+  Input laser scan.
+
+
+## Publishing Topic
+
+* `~output` (`geometry_msgs/PoseArray`)
+
+  Position of detected people.
+
+* `~output/markers` (`visualization_msgs/MarkerArray`)
+
+  MakerArray of detected people.
+
+## Parameters
+
+* `~queue_size` (`Int`, default: `1`)
+
+  Input queue size.
+
+* `~conf_thresh` (`Double`, default: `0.8`)
+
+  Threshold for confidence.
+
+* `~weight_file` (`String`, required)
+
+  Threshold for confidence.
```

Comment:
Description is correct? 
Maybe this param has checkpoint

Reply:
Thanks! I fixed the description
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2703/commits/3bc328a89cc21b04c7aa947e34dec442155a85c9